src/test/groovy/edu/ie3/simona/model/participant/ChpModelTest.groovy

```diff
@@ -90,7 +90,7 @@ class ChpModelTest extends Specification {
   }
 
   static def buildChpData(ChpState chpState, Double heatDemand) {
```

Comment:
Imho we should also change naming of buildChpData (and others with ChpData) accordingly.